metricproviders/datadog/datadog.go

```diff
@@ -300,14 +329,8 @@ func (p *Provider) parseResponseV2(metric v1alpha1.Metric, response *http.Respon
 	}
 
 	// Handle a populated query result
-	attributes := res.Data.Attributes
-	datapoint := attributes.Values[0]
-	timepoint := attributes.Times[len(attributes.Times)-1]
-	if timepoint == 0 {
-		return "", v1alpha1.AnalysisPhaseError, fmt.Errorf("Datapoint does not have a corresponding time value")
-	}
-
-	value := datapoint[len(datapoint)-1]
+	column := res.Data.Attributes.Columns[0]
+	value := column.Values[0]
 	status, err := evaluate.EvaluateResult(value, metric, p.logCtx)
 	return strconv.FormatFloat(value, 'f', -1, 64), status, err
```

Comment:
should we set  to 6 for precision? timeseries results seemed to be max 6 decimals